### DISCUSSION TITLE
x64 Task question

### DISCUSSION THREAD

**Discussion by mobotpsmocko** [qetza/replacetokens-task#64] [Q&A]
All of a sudden my deployments started failing with the following error:
A supported task execution handler was not found. The task does not carry an implementation that is compatible with your current operating system 'Windows(X64)'. Contact the task author for more details.

Has anyone experienced this? It looks like I am using v5 but cannot see where I can get v6 in marketplace to add to my devops server. If you have run into this and have a solution, pleas let me know.

**Comment by qetza** [comment]
Hi @mobotpsmocko,
All task versions are bundled in the same extension, you just have to change in the task name the version after the `@`.

As there was no release done on the task I'm not sure the issue is from it. Did you update the azure pipelines agent?

Could you share some more information on your setup? (agent version? are you using a self hosted pool? os? ...)

**Reply by mobotpsmocko** [reply]
Thanks for your help. 
The agent version is 2.153.1 running on self hosted windows server 2016 64 bit. I am not aware of any updates to the agents at this point. We had problems with another app deployment that uses .net core due to a sys admin updating the .net core runtime. I had him roll the machine (vm) back after we found out that it killed our deployments for that app. I swore I tested this app deployment after roll back and it worked. Then a few days later I tried to deploy this app and am getting this error. 

I did attempt to update agents before rolling back using azure devops portal and clicking update all agents button but it did not seem to do anything. I am at a total loss on how to fix the deployments at this point.

**Reply by mobotpsmocko** [reply]
Somehow is seems as if this is the only deployment agent running that version. All other apps deployment agents are running version 2.217.2. That is odd...

**Reply by mobotpsmocko** [reply]
Again, Thanks for your help. I was able to update to the 2.217.2 version and now the deployment works. At least this talk led me to the problem...

**Reply by qetza** [reply]
Glad you fixed your issue.
